Add new PR templates and revamp issue templates (#3711)
EssentialsX/Essentials: CONTRIBUTING.md

Contributing to EssentialsX
===========================

> Want to help improve EssentialsX? There are numerous ways you can contribute to the project.

Reviewer: ```suggestion
Want to help improve EssentialsX? There are several ways you can contribute to the project.
```

If you'd like to make a financial contribution to the project, you can join our [Patreon](https://www.patreon.com/essentialsx/),
or to make a one-off donation you can visit our [Ko-fi page](https://ko-fi.com/essentialsx). If you can't make a

> donation, don't worry! There's lots of other ways to contribute:

Reviewer 2: ```suggestion
donation, don't worry! There are lots of other ways to contribute:
```
Alternatively "There's a lot of..."

> donation, don't worry! There's lots of other ways to contribute:

Reviewer: Seems redundant
```suggestion
donation, don't worry!
```

Author: I prefer this as it introduces the next section. (also this text isn't new, it's been in README for a while)

Reviewer: Yeah this was minor. Just thought it could be cleaned up more since just above that, this section was introduced as a way of contributing.
